### DISCUSSION TITLE
How do I change the email address of an account? We don't have teams, and we only have one account, thanks T

### DISCUSSION THREAD

**Discussion by torbenanderson** [codemagic-ci-cd/codemagic-docs#1988] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1.

### Expected results

change email address of login account

### Actual results

new email address

**Comment by kulakarla** [comment, marked as answer]
Hello @torbenanderson 

This is unfortunately impossible currently.

You will have to create a new account with your desired email and add your Applications there. If you are using the Workflow Editor, you will have to reconfigure your apps. If you are using YAML, the transfer will be a bit simpler as the workflows are already written. In both cases, you will have to re-add your environment variables.

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
